Examples/Swift/FillPatternExample.swift

```diff
@@ -0,0 +1,86 @@
+import Mapbox
+
+@objc(FillPatternExample_Swift)
+
+class FillPattern_Swift: UIViewController, MGLMapViewDelegate {
+    var fillPatternImage: UIImage!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds)
+        
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.styleURL = MGLStyle.darkStyleURL(withVersion: MGLStyleDefaultVersion)
+        
+        // Set the map’s center coordinate and zoom level
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 38.849534447295895, longitude: -77.03922271728514), zoomLevel: 8.5, animated: false)
+        view.addSubview(mapView)
+        
+        // Set the UIImage to be used for the fill pattern
+        fillPatternImage = UIImage(named: "pattern")
+        
+        mapView.delegate = self
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Add the fill pattern image to used by the style layer
+        style.setImage(fillPatternImage.withRenderingMode(.alwaysTemplate), forName: "pattern")
+        
+        let coordinates = [
```

Comment:
This is fine, but it could be less verbose (in Swift, anyway) if the array were structured [like this one](https://github.com/mapbox/ios-sdk-examples/blob/dcb2167e60e356b3179a5a13d6799377d995933f/Examples/Swift/RuntimeAnimateLineExample.swift#L85-L161).